packages/recommendations-vdom/rollup.config.js

```diff
@@ -0,0 +1,27 @@
+import { plugins } from '../../rollup.base.config';
+
+import pkg from './package.json';
+
+if (!process.env.BUILD) {
+  throw new Error('The `BUILD` environment variable is required to build.');
+}
+
+const output = {
```

Comment:
I wonder if the output could also be exported from the base rollup config (as a function that takes pkg), it's the same for all of them

Reply:
Since we won't use rollup to build ESM I've inlined it in the part 2 of the migration, it's still different for the React packages since we use the `external` prop

Reply:
We plan to reiterate on the build scripts after @shortcuts stacked PR! (it'll include a fully-shared Rollup config, banners with license, etc.)